docs/command-reference/acl/dryrun.md

```diff
@@ -0,0 +1,29 @@
+---
+description: Simulate execution of a command from a user
+---
+
+# ACL DRYRUN
+
+## Syntax
+
+    ACL DRYRUN username command
```

Comment:
Just want to confirm if the behaviour is different from Redis.

Redis has something like:
- `ACL DRYRUN username command [arg [arg ...]]`
- i.e., `ACL DRYRUN VIRGINIA GET foo bar`